region/maridia/outer.json

```diff
@@ -1454,9 +1495,42 @@
                     "canBePatient",
                     {"obstaclesNotCleared": ["A"]}
                   ],
-                  "note": "Wait for the global crab to come around and freeze it to get up to the third ledge.",
+                  "note": "Wait for the global crab to come around and freeze it to get up to the second ledge on the right.",
                   "devNote": "The global crab will fall if the speed blocks are broken."
                 },
+                {
+                  "name": "Suitless Frozen Crab with HiJump",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "HiJump",
+                    "canTrickyUseFrozenEnemies",
+                    {"or": [
+                      "canCrouchJump",
+                      "canDownGrab"
+                    ]}
+                  ],
+                  "note": "Be sure not to get the first downward moving crab on screen until Samus is in position to quickly and accurately shoot and freeze it.",
+                  "devNote": "It can be done without a crouch jump or down grab, but is somewhat tight and unforgiving."
+                },
+                {
+                  "name": "Suitless Frozen Crab with Spring Ball",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "canSpringBallJumpMidAir",
+                    "canCrouchJump",
+                    "canTrickyUseFrozenEnemies",
+                    {"or": [
+                      "canTrickyJump",
+                      "canStationaryLateralMidAirMorph"
+                    ]}
+                  ],
+                  "note": [
+                    "Be sure not to get the first downward moving crab on screen until Samus is in position to quickly and accurately shoot and freeze it.",
+                    "The jump up to the platform with the fish can be done with a crouch jump and spring ball jump, but it is very tight. A stationary airball can help."
```

Comment:
airball isn't the right word - they can be purely vertical.  Its the diagonal input, frame perfect morph trick, that doesn't really have a name.